packages/manager/modules/sign-up/src/form/details/details.controller.js

```diff
@@ -82,8 +82,6 @@ export default class SignUpDetailsCtrl {
     if (get(this.signUpFormCtrl, 'rules.language.in.length', 0) === 1) {
       const [uniqueLanguage] = this.signUpFormCtrl.rules.language.in;
       this.signUpFormCtrl.model.language = uniqueLanguage.value;
-    } else {
```

Comment:
Can you use a boolean parameter to keep the behavior of emptying the language field when the user select a different country